site/app/controllers/MiscController.php

```diff
@@ -166,15 +167,15 @@ public function displayFile($dir = null, $path = null, $gradeable_id = null, $us
         $corrected_name = pathinfo($path, PATHINFO_DIRNAME) . "/" .  $file_name;
         $mime_type = mime_content_type($corrected_name);
         $file_type = FileUtils::getContentType($file_name);
-        if ($mime_type === "application/pdf" || (str_starts_with($mime_type, "image/") && $mime_type !== "image/svg+xml")) {
+        if (in_array($mime_type, FileUtils::Allowed_Inline_Types)) {
```

Comment:
Add `true` as a third argument to specify strict comparison. By default this is `false`.